include/deal.II/fe/fe_dgp_nonparametric.h

```diff
@@ -534,11 +534,13 @@ class FE_DGPNonparametric : public FiniteElement<dim, spacedim>
                                                                        spacedim>
       &output_data) const override;
 
+  using FiniteElement<dim, spacedim>::fill_fe_face_values;
```

Comment:
Just out of curiosity because I always forget the C++ rules on that topic: Does the code give a compile error without this using statement, or is it just that derived classes could not see the other interface any more to override it? In some sense, I would not feel too unhappy if none of the classes exposes the old interface anymore except the base class, as long as a user can *derive* from it.

Reply:
The problem is that the are two virtual functions with the same name in the base class. If you only override one of them you get a warning. This is the "beauty" of C++...